src/node/addon/package.json

```diff
@@ -19,6 +19,7 @@
     "debrid-link-api": "^1.0.1",
     "express-rate-limit": "^6.7.0",
     "ip": "^1.1.8",
+    "knightcrawler-utility": "file:../utility",
```

Comment:
Prefixing with the project name isn't really required as this utility package is never exposed (i.e. installable) to the outside world.